tests/test_standard_tags.py

```diff
@@ -256,6 +256,7 @@ def _setup_without_attack(self):
 
         self.requests_without_attack = {}
         for header, value in (self.FORWARD_HEADERS | self.FORWARD_HEADERS_VENDOR).items():
+            print(header)
```

Comment:
What are we printing the headers for?

Reply:
woops, it was for debugging https://github.com/DataDog/dd-trace-py/pull/8311
I'll remove it.